packages/openmrs/test/index.test.js

```diff
@@ -321,115 +291,49 @@ describe('fhir', () => {
     testServer
       .intercept({
         path: '/ws/rest/v1/patient',
-        query: { q: 'Sarah 1' },
-        method: 'GET',
+        method: 'DELETE'
       })
-      .reply(200, { results: [{ display: 'Sarah 1' }] }, { ...jsonHeaders });
-    const state = { configuration };
+      .reply(200, { results: 'Successful' }, {...jsonHeaders});
 
-    const { body } = await request(state, 'GET', '/ws/rest/v1/patient', {
-      query: { q: 'Sarah 1' },
-      baseUrl: state.configuration.instanceUrl,
-    });
-
-    expect(body.results[0].display).to.eql('Sarah 1');
-  });
-  it('should auto-fetch patients with a limit', async () => {
+    // Patient query with params interceptor
     testServer
       .intercept({
         path: '/ws/rest/v1/patient',
         query: { q: 'Sarah', limit: 1 },
         method: 'GET',
       })
-      .reply(
-        200,
-        {
-          results: [{ display: 'Sarah 1' }],
-          links: [
-            {
-              rel: 'next',
-              uri: 'https://fn.openmrs.org/ws/rest/v1/patient?q=Sarah&limit=1&startIndex=1',
-              resourceAlias: null,
-            },
-          ],
-        },
-        { ...jsonHeaders }
-      );
-    testServer
-      .intercept({
-        path: '/ws/rest/v1/patient',
-        query: { q: 'Sarah', limit: 1, startIndex: 1 },
-        method: 'GET',
-      })
-      .reply(
-        200,
-        {
-          results: [{ display: 'Sarah 2' }],
-        },
-        { ...jsonHeaders }
-      );
-
-    const state = { configuration };
-    const { body } = await request(state, 'GET', '/ws/rest/v1/patient', {
-      query: { q: 'Sarah', limit: 1 },
-      baseUrl: state.configuration.instanceUrl,
-    });
+      .reply(200, { results: testData.patientResults }, { ...jsonHeaders });
+  });
 
-    expect(body.results[0].display).to.eql('Sarah 1');
-    expect(body.results[1].display).to.eql('Sarah 2');
+  afterEach(() => {
+    // Remove all interceptors after each test
+    testServer.interceptors = [];
   });
 
-  it('should not auto-fetch if the user sets startIdex', async () => {
-    testServer
-      .intercept({
-        path: '/ws/rest/v1/patient',
-        query: { q: 'Sarah', limit: 1 },
-        method: 'GET',
-      })
-      .reply(
-        200,
-        {
-          results: [{ display: 'Sarah 1' }],
-          links: [
-            {
-              rel: 'next',
-              uri: 'https://fn.openmrs.org/ws/rest/v1/patient?q=Sarah&limit=1&startIndex=1',
-              resourceAlias: null,
-            },
-          ],
-        },
-        { ...jsonHeaders }
-      );
-    testServer
-      .intercept({
-        path: '/ws/rest/v1/patient',
-        query: { q: 'Sarah', limit: 1, startIndex: 1 },
-        method: 'GET',
-      })
-      .reply(
-        200,
-        {
-          results: [{ display: 'Sarah 2' }],
-        },
-        { ...jsonHeaders }
-      );
+  // Define state only once
+  const state = { configuration }
```

Comment:
Yep - we could probably do this once at the top of the file couldn't we? Because actually, despite what this comment says, by my count we define state 14 times :sweat_smile: